modules/roles_profiles/manifests/profiles/signing_users.pp

```diff
@@ -10,11 +10,12 @@
             # That is where the user virtual resources are generated
             require roles_profiles::profiles::users
             # Lookup the group array and realize their user resource
-            $users = lookup('user_groups.signing', Array, undef, undef)
+            $users = lookup('user_groups.signing', Array, undef, undef) - lookup('user_groups.relops', Array, undef, undef)
```

Comment:
Am I misreading, or will this result in 0 signing users?

Reply:
Oh, hang on, we're already creating the `relops` users so we'd be adding the `signing` users twice?

Reply:
[derail] While virtual resources are icky, in theory you should be able to realize them multiple times.  I would love to see stacktrace this causes, if you can simulate a failure, and see if we can eliminate it.
(In infra-puppet we gave up on virtual resources altogether, but I've stared at this class of problems a lot over time).